src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,377 @@
+
+The following steps describe how a PayID client can query a PayID server to obtain a PayID Easy Checkout URI Template. 
+
+### Step 1: Assemble PayID Easy Checkout Discovery URL
+The process of assembling a PayID Discovery URL is defined in section 4.1.1 of [PAYID-DISCOVERY][], and is
+the same as for PayID Easy Checkout Discovery.
+
+### Step 2: Query PayID Discovery URL
+A Webfinger query MUST be performed against the PayID Discovery URL assembled in the previous section,
+as described in section 4.2 of Webfinger.
+
+In response, the WebFinger resource returns a JSON Resource Descriptor (JRD)
+as the resource representation to convey information about the requested
+PayID.
+
+If the Webfinger endpoint returns a non-200 HTTP response status code, then PayID Easy Checkout is considered to 
+have failed. Clients SHOULD implement fallback measures to complete checkout in this case.
+
+### Step 3: Parse PayID Easy Checkout Metadata
+If a wallet supports PayID Easy Checkout, the PayID server MUST respond with a HTTP status code 200 and a JSON payload
+containing a JSON Resource Descriptor (JRD) as defined in section (TODO: link to jrd section) of this document. 
+The JRD MUST contain a link conforming to the link definition in section (TODO: link to section) of this paper.
+
+For example, a PayID server might respond to a PayID Easy Checkout Discovery query with the following payload:
+
+     {
+        "subject": "payid:alice$wallet.com",
+        "links": [
+            {
+                "rel" : "https://payid.org/ns/payid-easy-checkout/1.0",
+                "template": "https://wallet.com/checkout?amount={amount}&receiverPayId={receiverPayId}&currency={currency}&nextUrl={nextUrl}"
+            }
+        ]
+     }
+     
+The receiver client must parse this response to find the PayID Easy Checkout Link. 
+If the JRD returned from the PayID Easy Checkout Discovery query does not contain a 
+PayID Easy Checkout Link in its 'links' collection, PayID Easy Checkout is considered to have failed.
+Once a PayID Easy Checkout URI Template has been obtained from the PayID Easy Checkout Link by the PayID client, 
+PayID Easy Checkout Discovery is considered to be complete. 
+
+### Template Syntax
+This specification defines a simple template syntax for PayID Easy Checkout URI Template
+transformation.  A template is a string containing brace-enclosed
+("{}") variable names marking the parts of the string that are to be
+substituted by the corresponding variable values.
+
+This specification defines several variables, which MAY or MAY NOT be present in every PayID Easy Checkout URI Template.
+These variables are as follows:
+    
+    'amount': The amount that should be sent by the sender to the receiver
+    'receiverPayID': The PayID URI of the receiver
+    'currency': The ISO-4217 currency code that the sender should send
```

Comment:
My instinct is to call this `assetCode` instead because of our learnings from ILP -- it's possible that "payments" may not be made in a "currency" per-se (e.g., what if I want to pay in stock?).

In general, this is maybe an edge-case though, so happy to hear pushback.